test/functional/sites/emails_controller_test.rb

```diff
@@ -0,0 +1,23 @@
+require 'test_helper'
+
+class Sites::EmailsControllerTest < ActionController::TestCase
+
+  def setup
+    @provider  = FactoryBot.create(:provider_account)
+    @service = @provider.default_service
+    request.host = @provider.admin_domain
+    login_as(@provider.admins.first)
+  end
+
+  test 'show Service name sanitized' do
+    SERVICE_NAME_XSS = "API<script>alert('XSS')</script>"
+    SERVICE_NAME_XSS_SANITIZED = "APIalert('XSS')"
+    @service.name = SERVICE_NAME_XSS
```

Comment:
Move those constants outside of method body. Or create them as simple variable:
```suggestion
    service_name_xss = "API<script>alert('XSS')</script>"
        service_name_xss_sanitized = "APIalert('XSS')"
    @service.name =     service_name_xss
```